ci-operator/step-registry/openshift-qe/installer/bm/day2/cnv/openshift-qe-installer-bm-day2-cnv-commands.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+cat /etc/os-release
+
+if [ ${BAREMETAL} == "true" ]; then
+  bastion="$(cat /bm/address)"
+  # Copy over the kubeconfig
+  sshpass -p "$(cat /bm/login)" ssh -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null root@$bastion "cat ~/bm/kubeconfig" > /tmp/kubeconfig
+  # Setup socks proxy
+  sshpass -p "$(cat /bm/login)" ssh -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null root@$bastion -fNT -D 12345
+  export KUBECONFIG=/tmp/kubeconfig
+  export https_proxy=socks5://localhost:12345
+  export http_proxy=socks5://localhost:12345
+  oc --kubeconfig=/tmp/kubeconfig config set-cluster bm --proxy-url=socks5://localhost:12345
+fi
+
+oc config view
+oc projects
+
+# Install the CNV operator
+cat << EOF| oc apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: cnv-nightly-catalog-source
+  namespace: openshift-marketplace
+spec:
+  sourceType: grpc
+  image: quay.io/openshift-cnv/nightly-catalog:${CNV_VERSION}
+  displayName: OpenShift Virtualization Nightly Index
+  publisher: Red Hat
+  updateStrategy:
+    registryPoll:
+      interval: 8h
+EOF
+
+sleep 30
+
+STARTING_CSV=$(oc get packagemanifest -l catalog=cnv-nightly-catalog-source -n openshift-marketplace -o jsonpath="{$.items[?(@.metadata.name=='kubevirt-hyperconverged')].status.channels[?(@.name==\"nightly-${CNV_VERSION}\")].currentCSV}")
+
+cat << EOF| oc apply -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: openshift-cnv
+EOF
+
+cat << EOF| oc apply -f -
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+    name: kubevirt-hyperconverged-group
+    namespace: openshift-cnv
+spec:
+    targetNamespaces:
+    - openshift-cnv
+EOF
+
+cat << EOF| oc apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+    name: hco-operatorhub
+    namespace: openshift-cnv
+spec:
+    source: cnv-nightly-catalog-source
+    sourceNamespace: openshift-marketplace
+    name: kubevirt-hyperconverged
+    startingCSV: ${STARTING_CSV}
+    channel: "nightly-${CNV_VERSION}"
+EOF
+
+until oc get csv -n openshift-cnv $STARTING_CSV ; do  sleep 5; done
+oc wait --timeout=300s -n openshift-cnv csv $STARTING_CSV --for=jsonpath='{.status.phase}'=Succeeded
+
+cat << EOF| oc apply -f -
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+  namespace: openshift-cnv
+Spec: {}
+EOF
+
+sleep 20
```

Comment:
Do we want to use infra nodes to host CNV stuff?

Reply:
which aspects? Does the product mention to do that?

Reply:
@jeniferh wdyt? is it worth it? 
```Optional: Configure Infra and Workloads node placement options for OpenShift Virtualization components.```

I don't see what they recommend moving. Things like the cni-linux-bridge, virt handler, bridge-marker, etc are per-node. Is the virtcontroller/virt-apiserver heavy components?

Reply:
Virt control plane load shouldn't be much of a concern unless you were just constantly churning VM creation, even then it tends to run on beefy baremetal nodes, don't really see Infra usage in practice.